src/App.css

```diff
@@ -0,0 +1,723 @@
+.App {
+  text-align: center;
+}
+
+.App-header {
+  background-color: var(--wrapper-background-color);
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+/*------------------------------Wrapper---------------------------------*/
+
+.wrapper {
+  display: grid;
+  position: relative;
+  grid-template-columns: 15% 73% 12%;
+  grid-template-rows: 15% 75% 10%;
+  text-decoration: 1px solid #ffffff;
+  width: 100vw;
+  height: 100vh;
+  background-color: var(--wrapper-background-color);
+  overflow: hidden;
+}
+
+.header,
+.nav,
+.section,
+.aside,
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+/*------------------------------Header---------------------------------*/
+
+.header {
+  position: relative;
+  background: var(--background-color-header);
+  grid-column: 1/4;
+  grid-row: 1;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 1.2em;
+  color: white;
+}
+
+.header-logo {
+  position: absolute;
+  left: 4.2em;
```

Comment:
Why is the header-logo absolute? When possible, absolutes should be avoided. We can adjust its position using flexbox features. In this case `flex-grow: 1`, to make the element fill the entire area and add a little spacing with `margin-right`. [screenshot](https://user-images.githubusercontent.com/14869087/120864649-738e6980-c584-11eb-91ec-af8c7267d26e.png)

Finally to properly center it, you need to create an extra container with `display: inline-block` to force it to just fill the area needed and then align it to the center. 
[screenshot](https://user-images.githubusercontent.com/14869087/120864888-e0a1ff00-c584-11eb-9d5c-d80f628a9825.png)